woocommerce/woocommerce
# add/a2p data api-core-tests

### All Submissions:






### Changes proposed in this Pull Request:
Update playwright api-core-tests for shipping crud operations


Closes 417-gh-woocommerce/woocommerce-quality.






### How to test the changes in this Pull Request:



1. Pull down the code
2. Navigate to woocommerce/plugins/woocommerce
3. Execute tests with pnpm test:api-pw
4. All tests including the new data tests should execute and pass successfully



### Other information:




### FOR PR REVIEWER ONLY:
